### PR TITLE
added README and 4 folders

### PR DESCRIPTION
**Pull request by marianalevineu**
Added training material - initial commit.

### CODE REVIEW

**Review by joseph-cooke — APPROVED** [review]

**Comment by dhml** [comment]
@marianalevineu, @ShobanaSekar, @manasvita, it would be prudent to include the Systems team as reviewers on pull requests for this particular training.  Thanks!

**Comment by marianalevineu** [comment]
@dhml sorry, we assumed you guys were busy with other stuff, but if you do wish to review it please feel free thanks!

**Comment by dhml** [comment]
> @dhml sorry, we assumed you guys were busy with other stuff, but if you do wish to review it please feel free thanks!

@marianalevineu, no worries this time as it looks fine, just for future consideration.

File: Exercise_1/divide_and_split_files.bash [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+echo "list of files in directory $PWD before split command:"
+ls 
+split mobydick.txt
+echo "list of files in directory $PWD after split command:"
+ls
+
+# Check how many rows each file has with the “wc -l” utility:
+echo "Number of lines in file mobydick.txt:"
+wc -l mobydick.txt
+echo "How many rows each of the remainig files has:"
```
Comment (ShobanaSekar) [inline Exercise_1/divide_and_split_files.bash:26]: remaining

File: Exercise_2/myscript_batch.bash [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#SBATCH --time=00:10:00
+#SBATCH --mem=2GB
+#SBATCH --output=ex2.out
+#SBATCH --error=ex2.err
+
+## This script demonstrates basic automation in Shell using a Shell script. We will create a simple script that creates files with different contents:
+
+## use the mobydick.txt file from Exercise 1 to select all lines that contain the word "Mountains". Use the '>' operator to print the output lines into a new file:
+grep "Mountains" ../Exercise_1/mobydick.txt > mobydick.Lines-with-Mountains.txt
+#grep "Mountains" ~/training-linux-shell-scripting/Exercise_1/mobydick.txt > mobydick.Lines-with-Mountains.txt
```
Comment (ShobanaSekar) [inline Exercise_2/myscript_batch.bash:15]: can remove this commented line?

File: Exercise_3/file_managment_batch.bash [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+#SBATCH --partition=express
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --job-name=file_managment
+#SBATCH --time=00:10:00
+#SBATCH --mem=2GB
+#SBATCH --output=ex3.out
+#SBATCH --error=ex3.err
+
+## This script demonstrates basic file managment autmation. The script will create 4 directories, and will create 4 files inside those directories.
```
Comment (ShobanaSekar) [inline Exercise_3/file_managment_batch.bash:11]: management automation

File: Exercise_4/file_row_processing.bash [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+rows_file=rows.txt
+output_file=post_processing_file.txt
+
+# Define the number of rows to process from a command-line argument:
+row_num=$1
+# Evaluate the number of rows in the rows file using utility `wc -l` , then pipe the 
+# output to the 'awk' utility which will print the first argument:
+rows_file_length=`wc -l ${rows_file} | awk '{print $1}'` 
+
+# Check if $row_num is a valid number (using use a regular expression) and smaller than number of lines in $rows_file:
```
Comment (ShobanaSekar) [inline Exercise_4/file_row_processing.bash:28]: using a regular expression

**Comment by dhml** [comment]
@ShobanaSekar, thanks for the additional comments.  Note that any further edits will require a new pull request as this one has already been merged.  @marianalevineu, @manasvita, @joseph-cooke, FYI.

**Comment by marianalevineu** [comment]
thanks @ShobanaSekar and @dhml  I've applied those changes in a new branch.

**Pull request merged**